qforce/fragment.py

```diff
@@ -32,13 +32,16 @@ def fragment(mol, qm, job, config):
 
         if name not in unique_dihedrals:
             unique_dihedrals[name] = term.atomids
-
+    generated = [] # Number of fragments generated but not computed
     for name, atomids in unique_dihedrals.items():
         frag = Fragment(job, config, mol, qm, atomids, name)
         if frag.has_data:
             fragments.append(frag)
+        if frag.has_inp:
```

Comment:
elif config.batch_run and frag.has_inp:

Reply:
The config here doesn't have the batch_run attribute.

Reply:
hmm I think it's the general config. Then you need: config.scan.batch_run